tableaudocumentapi/workbook.py

```diff
@@ -11,25 +11,23 @@
 
 
 class Workbook(object):
-    """
-    A class for writing Tableau workbook files.
-
-    """
+    """A class for writing Tableau workbook files."""
 
     ###########################################################################
```

Comment:
so are we leaving this comment in to denote the public API? fine if we do just checking